src/hotspot/share/gc/parallel/psScavenge.cpp

```diff
@@ -180,55 +180,64 @@ class PSKeepAliveClosure: public OopClosure {
 class PSEvacuateFollowersClosure: public VoidClosure {
  private:
   PSPromotionManager* _promotion_manager;
+  TaskTerminator* _maybe_terminator;
+  uint _worker_id;
+
  public:
-  PSEvacuateFollowersClosure(PSPromotionManager* pm) : _promotion_manager(pm) {}
+  PSEvacuateFollowersClosure(PSPromotionManager* pm, TaskTerminator* maybe_terminator, uint worker_id)
+    : _promotion_manager(pm), _maybe_terminator(maybe_terminator), _worker_id(worker_id) {}
 
   virtual void do_void() {
     assert(_promotion_manager != NULL, "Sanity");
     _promotion_manager->drain_stacks(true);
     guarantee(_promotion_manager->stacks_empty(),
               "stacks should be empty at this point");
-  }
-};
 
-class PSRefProcTaskExecutor: public AbstractRefProcTaskExecutor {
-  virtual void execute(ProcessTask& process_task, uint ergo_workers);
+    if (_maybe_terminator != nullptr) {
```

Comment:
Please change the `!= NULL` to `!= nullptr` at least in the same method too (above). It is not required to fix it everywhere (in this file) for this change, but using different styles in the same method is weird.

Reply:
Okay.